server/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java

```diff
@@ -117,6 +117,15 @@
         return concreteIndexNames(context, indexExpressions);
     }
 
+    public List<String> dataStreamNames(ClusterState state, IndicesOptions options, String... indexExpressions) {
```

Comment:
Maybe add a unit test for this method to `IndexNameExpressionResolverTests`?

Reply:
Done